### DISCUSSION TITLE
Topgrade Available on Fedora Copr

### DISCUSSION THREAD

**Discussion by li-lay** [topgrade-rs/topgrade#1003] [General]
Hey guys, recently I've decided to package Topgrade into rpm package. 
And I have built it on Fedora Copr  and only for Fedora 41 as of now.
You can enable the Copr repo and try it out. Maybe give me some feedback! :)

https://copr.fedorainfracloud.org/coprs/lilay/topgrade/

I'm planning on summit a review request to get this into Fedora repo. But I'm stilling learning/reading about it right now.

**Comment by SteveLauC** [comment]
Thanks for doing it! As a Fedora user, it always bothers me not to have Topgrade packaged for Fedora 😮‍💨

**Reply by li-lay** [reply]
Glad to know that Maintainer also a Fedora user 😅 ! Maybe we can get this inside Readme so others can also get `topgrade` via Copr on Fedora.

**Comment by li-lay** [comment]
Here's more update on the Review Request of the package if anyone interested.
https://bugzilla.redhat.com/show_bug.cgi?id=2331910

**Reply by SteveLauC** [reply]
I am not familiar with this procedure, but if you need any help on Topgrade's side, just ping me:)

**Comment by li-lay** [comment]
Sorry for my absent for the past few months. I've been busy transferring to a new University.
Update on the topgrade Copr, now have successfully built 16.0.3.

@SteveLauC And about my trying to get topgrade into official fedora package repo. Failed! But I'll try again haha!

**Reply by SteveLauC** [reply]
Thanks for the effort you put into this!
